pages/emailSender.tsx

```diff
@@ -550,6 +570,13 @@ const EmailSender: NextPage = () => {
             </StyledResultMessage>
           </SectionContainer>
           <StyledFinalMessagesContainer>
+            {useSignature && signatureData && (
+              <>
+                <StyledDivider />
+                <Typography>Email signature preview:</Typography>
```

Comment:
IMO the email previews should contain the signature as well, or people may have duplication (including it in their email content and then accidentally also adding this in). Unsure if you take this suggestion if we also still require this email signature preview and where it is best placed, though I still do potentially see the value

Reply:
Thought about it more - one more option is to explicitly make a full signature step separate from the content step, where they can either add a HBP-style signature or a more casual textual signature, but that could also be an unnecessary step in user flow. Feel free to ignore this, because I'm not sure if either idea adds value

Reply:
I decided to include the signature in each email preview 👍